src/content/self-hosted/install/releases.mdx

```diff
@@ -11,7 +11,7 @@ id: releases
 ### Bugfixes
 
 - Upgraded the Okteto CLI version to [2.14.3](https://github.com/okteto/okteto/releases/tag/2.14.3).
-  - The Global Push feature, which was introduced in version 2.14.0, is experiencing instability and will therefore be disabled.
+  - Images weren't being built for the same commit even if they were using different build arguments
```

Comment:
I would mention something in this line:
Okteto CLI 2.14.0 introduced an optimization to build only once the image for a given commit. Okteto CLI 2.14.3 disables this optimization while we fix some inconsistencies around its behavior